### PR TITLE
Added read support for .mzdata chromatograms

### LINKED ISSUE #579: mzData Chromatogram Read Support
https://www.psidev.info/mass-spectrometry-workgroup#mzdata by @HUPO-PSI is a deprecated open standard file format. I found test data for it at https://zenodo.org/record/3631074

### PR DESCRIPTION
**Pull request by Mailaender**
Closes https://github.com/eclipse/chemclipse/issues/579

### CODE REVIEW

File: chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java [hunk 1]
```diff
@@ -1,40 +1,73 @@
 /*******************************************************************************
- * Copyright (c) 2014, 2018 Lablicate GmbH.
```
Comment (eselmeister) [inline chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java:2]: Why did you remove the copyright header instead of extending it?

File: chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java [hunk 2]
```diff
@@ -1,40 +1,73 @@
...
+ * Copyright (c) 2021 Lablicate GmbH.
  * 
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
  * 
  * Contributors:
- * Dr. Philip Wenig - initial API and implementation
+ * Matthias Mailänder - initial API and implementation
```
Comment (eselmeister) [inline chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java]: Same as above?

File: chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/converter/ChromatogramImportConverter.java [hunk 3]
```diff
@@ -8,35 +8,65 @@
  *
```
Comment (eselmeister) [inline chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/converter/ChromatogramImportConverter.java:8]: Adjust the year to 2021.
Add yourself to the committer list if relevant parts of the code have been edited by yourself.

**Comment by eselmeister** [comment]
@Mailaender Please have a look at the above comments.

File: chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java (hunk 1 above)
Comment (Mailaender) [reply chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java:2]: I rewrote the whole file from scratch.

File: chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/converter/ChromatogramImportConverter.java (hunk 3 above)
Comment (Mailaender) [reply chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/converter/ChromatogramImportConverter.java:8]: This is just copy and pasted code. Nothing is from me or 2021.

File: chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java (hunk 2 above)
Comment (Mailaender) [reply chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java]: Everything in this file was replaced. Not even the implemented methods are the same.

File: chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java (hunk 1 above)
Comment (eselmeister) [reply chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java:2]: Files shall inherit the header of the original, see comment from Wayne Beaton:
https://github.com/eclipse/chemclipse/issues/407

`"The short version is that when you split a file during a refactoring exercise, new files automatically inherit the header of the original. If even a fragment of a single line of the original content survives the refactoring (whether generated, rote, or otherwise), then the original copyright statement and author information should be retained."`

File: chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java (hunk 2 above)
Comment (eselmeister) [reply chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java]: See header comment above.

File: chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/converter/ChromatogramImportConverter.java (hunk 3 above)
Comment (eselmeister) [reply chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/converter/ChromatogramImportConverter.java:8]: The change (copying) has been made in 2021, so the header needs to be updated.

**Comment by Mailaender** [comment]
Updated the headers.

File: chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java [hunk 4]
```diff
@@ -8,33 +8,67 @@
  *
```
Comment (eselmeister) [inline chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/src/org/eclipse/chemclipse/msd/converter/supplier/mzdata/io/ChromatogramReader.java:8]: Update the year to 2021.

File: chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -13,7 +13,8 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.chemclipse.processing;bundle-version="0.8.0",
  org.eclipse.chemclipse.converter;bundle-version="0.8.0",
  org.eclipse.chemclipse.model;bundle-version="0.8.0",
- org.eclipse.chemclipse.support;bundle-version="0.8.0"
+ org.eclipse.chemclipse.support;bundle-version="0.8.0",
```
Comment (eselmeister) [inline chemclipse/plugins/org.eclipse.chemclipse.msd.converter.supplier.mzdata/META-INF/MANIFEST.MF:16]: org.eclipse.chemclipse.xxd.converter.supplier.chemclipse
Whenever possible, set the bundle-version.

**Comment by eselmeister** [comment]
@Mailaender The implementation looks good. I have added two more comments.

**Comment by Mailaender** [comment]
Updated again.
